vignettes/email-overview.Rmd

```diff
@@ -0,0 +1,141 @@
+---
+title: "Überblick Open-Access-Metadaten-Schnelltest"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Überblick Open-Access-Metadaten-Schnelltest}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+Überblick Open-Access-Metadaten-Schnelltest
```

Comment:
we should stick to ATX Headings as recommended by most markdown styleguides https://google.github.io/styleguide/docguide/style.html